src/NServiceBus.RabbitMQ/Connection/RabbitMqConnectionFactory.cs

```diff
@@ -44,10 +44,9 @@ public RabbitMqConnectionFactory(ConnectionConfiguration connectionConfiguration
             };
         }
 
-        public virtual IConnection CreateConnection(string purpose)
+        public virtual IConnection CreateConnection(string connectionName)
```

Comment:
What is the compelling reason to make this changes? Does it justify breaking semver?